db/versions/0048.yml

```diff
@@ -0,0 +1,44 @@
+version: 48
+description: Add an index on state in the workers table
+migrationScript: |-
+  begin
+    create index workers_state_idx on workers (state);
+  end
+downgradeScript: |-
+  begin
```

Comment:
should probably drop the index here, otherwise an upgrade -> downgrade -> upgrade again cycle will fail with an error that the index already exists.

Reply:
Out of curiosity, do we know why we never dropped an index in `downgradeScript` before?

Reply:
I suspect previous indexes have been introduced at the same time as the table they index, and dropping a table automatically drops associated indexes.  I think this is the first time that we've added an index in a version different from that where the table was added.  But, that's just a hunch -- I didn't check.

Reply:
That would make sense. Thanks!